src/NzbDrone.Core/Tv/ShouldRefreshSeries.cs

```diff
@@ -28,9 +28,9 @@ public bool ShouldRefresh(Series series)
                 return true;
             }
 
-            if (series.LastInfoSync >= DateTime.UtcNow.AddHours(-6))
+            if (series.LastInfoSync >= DateTime.UtcNow.AddHours(-1))
```

Comment:
No reason to do this. You're not using this interface in the single refresh since you added the if explicitly.
There's also no reason to lower this threshold for all series.

Reply:
My thinking was, if I updated a series on TVDB and refreshed not knowing about the various caches nothing would update, then if a couple hours later Sonarr refreshed it automatically it'd skip the update it and not update automatically for about 15 hours.

Reverted it though.